debian/build.go

```diff
@@ -356,7 +356,7 @@ func getKubeletCNIVersion(v version) (string, error) {
 	}
 
 	if sv.GTE(v190) {
-		return fmt.Sprintf("= %s", cniVersion), nil
+		return fmt.Sprintf(">= %s", cniVersion), nil
 	}
 	return fmt.Sprint("= 0.5.1"), nil
```

Comment:
should this also get updated to be `>=` not sure if it matters or not?

Reply:
it's for 1.8 which is way outside of the support skew.